NickvisionTagger.GNOME/Program.cs

```diff
@@ -37,18 +37,19 @@ public Program()
             @"* Added offset field to synchronized lyrics page 
               * Added the ability to import and export LRC files in the synchronized lyrics page
               * Fixed an issue where synchronized lyrics stored in the LRC format were not displayed correctly
+              * Fixed an issue where using WebP image for album art caused crash
```

Comment:
should be `caused a crash` and then needs to be added to metainfo xml file as well